come-lavoriamo/consuntivazione.md

```diff
@@ -1,62 +1,89 @@
-ognuno può, se lo ritiene opportuno, anticipare una giornata su un cliente e recuperare la giornata
-dell'altro cliente in un'altro momento (comunicandolo :)).
-
-Il tema della valutazione delle ore lavorative è delicato e l'interpretazione spesso non è univoca.
-La soluzione che adottiamo a Nebulab è quella di segnare il tempo effettivamente svolto sul progetto
-avendo come unità minima la mezz'ora. Quali attività include e quali non il "tempo effettivamente
-svolto"?
-
-Include:
-
-- Tempo di implementazione effettiva
-- Tempo di pair-programming sullo stesso progetto
-- Tempo di comunicazione direttamente correlato con il lavoro che si sta svolgendo
-- Tempo di studio direttamente ed esclusivamente correlato con il lavoro che si sta svolgendo
-- Tempo di pausa normale (<30 minuti al giorno)
-
-Non include:
-
-- Tempo di implementazione su altri progetti
-- Tempo di pair-programming su altri progetti
-- Tempo di comunicazione non correlati con il lavoro che si sta svolgendo
-- Tempo di studio non direttamente e non esclusivamente correlato con il lavoro che si sta svolgendo
-- Tempo di pausa lunghi (>30 minuti al giorno)
-
-Come potete immaginare esistono diverse zone d'ombra nel confine tra attivita incluse e attività non
-incluse. La raccomandazione è quella di usare il buonsenso e di chiedere una mano ad un'altra
-persona, magari che è sul progetto con voi, se vi trovate in difficoltà nella determinazione del
-numero di ore svolte.
-
-Un piccolo trucco che potete utilizzare è cercare di quantificare le attività non incluse e
-sottrarle alle ore di presenza in ufficio.
-
-Un caso particolare: I tempi di attesa delle risposte del cliente che passo girandomi i pollici
+# Misurare il tempo
+
+In questa sezione vogliamo offrire uno spunto per la corretta determinazione da un lato delle ore
+lavorative spese sui diversi progetti e dall'altro delle ore di presenza in ufficio.
+
+### Le ore lavorative
+
+L'effort su un progetto viene individuato con un numero di ore, o più semplicemente di giornate,
+settimanali da spendere sul progetto stesso che é importante conoscere e tenere presente sempre. Se
+avete dubbi sulla quantità di ore da svolgere su un progetto a voi assegnato non esitate a chiederlo
+al vostro mentore.
+
+> Se salto una giornata su un progetto di un cliente devo [recuperare il venerdì?](https://github.com/nebulab/playbook/blob/master/crescita-personale/venerdi.md#lavorare-sui-clienti-il-venerd%C3%AC)
+
+Il tempo svolto effettivamente sul progetto deve essere poi misurato. Adottiamo un time tracker per
+diversi motivi:
+
+- onestà e trasparenza;
+- efficenza amministrativa;
+- facilità di utilizzo;
+- precisione dei dati.
+
+É fondamentale tenere presente che l'attività che svolgiamo è soprattutto creativa. Non poniamoci
+degli obiettivi in termini di ore da svolgere e cerchiamo sempre di dare la precedenza alla qualità
+del lavoro. Lasciamo che il tracker sia un semplice strumento per la consuntivazione delle ore, non
```

Comment:
"consuntivazione" is a bit complex, can we go with "tracciamento" or something similar?

Reply:
going with "strumento per tracciare le ore"